internal/slack_client/slack.go

```diff
@@ -39,8 +38,8 @@ type Conversation struct {
 func NewSlack(cfg config.Config) ClientForSlack {
 	return &slackStruct{
 		log:             cfg.Log(),
-		userAdminClient: slack.New(os.Getenv("USER_ADMIN_TOKEN")),
-		superBotClient:  slack.New(os.Getenv("SUPER_BOT_TOKEN")),
+		userAdminClient: slack.New(os.Getenv("USER_TOKEN")),
+		superBotClient:  slack.New(os.Getenv("BOT_TOKEN")),
```

Comment:
may be just bot token and user token :)

Reply:
Also it will be better to put `os.Getenv(...)` in config package, get it there, validate and throw error if no token specified, because now your we don't know what will happen if no token would be passed

Reply:
Already added Slack config in another branch, but at the moment it parses tokens from `config.yaml`. Will rewrite it to use env vars ✅